#!/usr/bin/env python
PACKAGE = "vision_module"

from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 
gen.add("NUM_CAND", int_t, 0, "NUMBER OF CANDIDATES", 3, 1, 10)
gen.add("PUB_FEATURES", bool_t, 0, "PUBLISH OBJECT FEATURES", False)
gen.add("DISPLAY", bool_t, 0, "DISPLAY RESULT", False)
gen.add("SEND_IMAGE", bool_t, 0, "SAVE IMAGE", False)
gen.add("SAVE_IMAGE", bool_t, 0, "SAVE IMAGE", False)

exit(gen.generate(PACKAGE, "object_recognition", "object_recognition"))
